Mathlib/NumberTheory/Zsqrtd/Basic.lean

```diff
@@ -273,18 +273,18 @@ instance nontrivial : Nontrivial (ℤ√d) :=
   ⟨⟨0, 1, (Zsqrtd.ext_iff 0 1).not.mpr (by simp)⟩⟩
 
 @[simp]
-theorem coe_nat_re (n : ℕ) : (n : ℤ√d).re = n :=
+theorem re_natCast (n : ℕ) : (n : ℤ√d).re = n :=
```

Comment:
This lemma name is inconsistent with the `ofNat` one below. Can you either revert this order change, or spin off a PR that fixes all the names in this file?